src/System.IO.Pipelines/src/System/IO/Pipelines/ThreadPoolScheduler.NetStandard20.cs

```diff
@@ -9,26 +9,19 @@ namespace System.IO.Pipelines
 {
     internal sealed class ThreadPoolScheduler : PipeScheduler
     {
-        public override void Schedule(Action action)
+        public override void Schedule<TState>(Action<TState> action, TState state)
         {
-            System.Threading.ThreadPool.QueueUserWorkItem(_actionWaitCallback, action);
+            System.Threading.ThreadPool.QueueUserWorkItem(_actionObjectWaitCallback, new ActionObjectAsWaitCallback<TState>(action, state));
         }
 
-        public override void Schedule(Action<object> action, object state)
-        {
-            System.Threading.ThreadPool.QueueUserWorkItem(_actionObjectWaitCallback, new ActionObjectAsWaitCallback(action, state));
-        }
-
-        private readonly static WaitCallback _actionWaitCallback = state => ((Action)state)();
-
-        private readonly static WaitCallback _actionObjectWaitCallback = state => ((ActionObjectAsWaitCallback)state).Run();
+        private readonly static WaitCallback _actionObjectWaitCallback = state => ((ActionObjectAsWaitCallback<TState>)state).Run();
```

Comment:
Nit: across corefx the style is to use an s_ prefix for non-public static fields.